packages/material-ui/src/Slider/Slider.js

```diff
@@ -405,6 +405,14 @@ const Slider = React.forwardRef(function Slider(props, ref) {
     name: 'Slider',
   });
 
+  const onChange =
```

Comment:
```suggestion
  const handleChange =
```

Reply:
I was hoping to avoid having to change the other parts of the code where `onChange` is called, that's why I defined a local function called `onChange` which called the function from props instead. Is this the pattern you prefer?

Reply:
It seems to be a convention in the codebase, hence my proposal.